crates/ruff/src/settings/types.rs

```diff
@@ -212,6 +212,8 @@ impl FromStr for PatternPrefixPair {
 #[serde(rename_all = "kebab-case")]
 #[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
 pub enum SerializationFormat {
+    Short,
+    #[cfg_attr(feature = "clap", value(alias("Short"), hide(true)))]
```

Comment:
I don't know if we should hide it or not. Ideally, there would be a way to deprecate it but I didn't figure out a way to do that.

Reply:
Ideally, for me: hide it + log a warning on-use.